shared_model/validators/field_validator.cpp

```diff
@@ -365,6 +365,17 @@ namespace shared_model {
       }
     }
 
+    void FieldValidator::validateNonNegativeHeight(
```

Comment:
I don't think this method is necessary, since height cannot be negative due to its type.